Modix.Data/Models/Core/MessageCountPerChannel.cs

```diff
@@ -2,13 +2,8 @@
 {
     public class MessageCountPerChannel
     {
-        public ulong ChannelId { get; }
-        public string ChannelName { get; }
-
-        public MessageCountPerChannel(ulong channelId, string channelName)
-        {
-            ChannelId = channelId;
-            ChannelName = channelName;
-        }
+        public ulong ChannelId { get; set; }
+        public string ChannelName { get; set; } = null;
```

Comment:
```suggestion
        public string ChannelName { get; set; } = null!;
```
+null-forgiving operator, so it doesn't warn about trying to assign null to a non-nullable property